.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Ensure that there is no accidental exposure of these tokens/APIs in logs or error messages; implement appropriate logging practices in the application that utilizes them.